resources/views/applicant/job_post/apply.html.twig

```diff
@@ -57,24 +57,28 @@
 
     </div>
 
-    {# Tristan, we need to check for the user's log state to display the appropriate button here. #}
-
     <div
         class="job-post__button-wrapper">
 
-        <a
+        {% if not job.isOpen() %}
```

Comment:
@tristan-orourke if you take a look at [my PR for previewing Job Posters](https://github.com/GCTC-NTGC/TalentCloud/pull/614/files#diff-a6fec36e1f2745cb1f4d19ec2dc4f4e2R132) I tried to move the logic for the Apply Button back into the Controller for the sake of more concise templating.